### PR TITLE
[tosa] Support for Aten[Unsqueeze|Contiguous|Dropout|Reshape|View] ops

### PR DESCRIPTION
**Pull request by anupgangwar**
Signed-off-by: Anup Gangwar <anup.gangwar@arm.com>

### CODE REVIEW

**Comment by anupgangwar** [comment]
Tagging @cathyzhyi for review.

File: lib/Conversion/TorchToTosa/TorchToTosa.cpp [hunk 1]
```diff
@@ -2329,6 +2329,118 @@ LogicalResult ConvertAtenOp<AtenThresholdOp>::matchAndRewrite(
...
+    ConversionPatternRewriter &rewriter) const {
+
+  // Not a tensor type.
+  auto selfType = adaptor.input().getType().dyn_cast<TensorType>();
+  if (!selfType)
+    return op.emitError("Only tensor types are currently supported");
+
+  // FIXME: train and p are not handled.
+
+  if (!op.train().getType().isa<Torch::NoneType>()) {
```
Comment (cathyzhyi) [inline lib/Conversion/TorchToTosa/TorchToTosa.cpp:2402]: the `if` is not needed because `train` can only be bool type.

**Review by cathyzhyi — APPROVED** [review]
looks mostly good except one if.

File: lib/Conversion/TorchToTosa/TorchToTosa.cpp (hunk 1 above)
Comment (anupgangwar) [reply lib/Conversion/TorchToTosa/TorchToTosa.cpp]: Incorporated in the latest push.

**Pull request merged**